.github/workflows/ci.yml

```diff
@@ -262,5 +262,7 @@ jobs:
         run: py -3 test\util\rpcauth-test.py
 
       - name: Run functional tests
+        env:
+          TEST_RUNNER_EXTRA: ${{ github.event_name != 'pull_request' && '--extended --exclude feature_dbcrash' || '' }}
```

Comment:
Are we OK with an increased runtime:
![image](https://github.com/bitcoin/bitcoin/assets/32963518/0b43f513-f4f5-49dc-bd3d-f782c2498257)
?

Reply:
```
feature_dbcrash.py                                     | ✓ Passed  | 7086 s
```

Reply:
It takes one runner busy approximately twice longer than other runners. Of course, not a big deal, considering we are provided with 20 runners.

Reply:
> If there are any downsides, you can skip the functional tests on the master branch completely on `job_id % n == 0`, where `n` is the down-scaling factor, but then run them completely otherwise?

Let's left it as is and see how it will go.